packages/plugin-feature/test/integration/spec/feature.js

```diff
@@ -45,4 +45,37 @@ describe(`plugin-feature`, function() {
       });
     });
   });
+
+  describe(`#setBatchUserFeatures()`, () => {
+    before(() => testUsers.create({count: 1})
+      .then((users) => {
+        spock = users[0];
+        spark = new CiscoSpark({
+          credentials: {
+            authorization: spock.token
+          }
+        });
+        return spark.device.register();
+      })
+    );
+
+    const featureUpdateArray = [{
+      key: `key1`,
+      val: `value1`,
+      type: `USER`,
+      mutable: `true`
+    }, {
+      key: `key2`,
+      val: `value2`,
+      type: `USER`,
+      mutable: `false`
+    }];
+
+    it(`sets a value for two user feature toggle`, () => {
+      return spark.feature.setBatchUserFeatures(featureUpdateArray)
```

Comment:
I believe setFeature also calls device.refresh(), thus ensuring we've updated our local feature cache. This test should spy on device.refresh to make sure it gets called. 

You'll want a similar integration test that does the full round trip to make sure the api actually does what we expect.